snap/snapcraft.yaml

```diff
@@ -37,8 +30,8 @@ parts:
     prime:
       - jetbrains-studio.desktop
   android-studio:
-    after: [gnome]
-    plugin: nil
+    plugin: dump
```

Comment:
please change that back to `nil`

Reply:
We need to use `dump` to access the `get_latest.py` file.

Reply:
the `source: .` stanza will help with that. We dont need to use dump.

Reply:
I have changed that :smile: :smile: